internal/sirius/document.go

```diff
@@ -0,0 +1,121 @@
+package sirius
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+type Document struct {
+	ID              int    `json:"id,omitempty"`
+	CorrespondentID int    `json:"correspondentID"`
+	Type            string `json:"type"`
+	SystemType      string `json:"systemType"`
+	FileName        string `json:"fileName,omitempty"`
+	Content         string `json:"content"`
+}
+
+type documentTemplateApiResponse map[string]json.RawMessage
+type insertApiResponse map[string]json.RawMessage
+
+type UniversalTemplateData struct {
+	Location        string `json:"location"`
+	OnScreenSummary string `json:"onScreenSummary"`
+}
+
+type DocumentTemplateApiData struct {
```

Comment:
I don't think this is ever used externally, so it can be excluded from export to avoid confusion:
```suggestion
type documentTemplateApiData struct {
```